src/components/SignUpForm/styles.scss

```diff
@@ -0,0 +1,37 @@
+@import "../../styles/main.scss";
+
+.container {
+    min-height: 100vh;
+    @include flex-center();
+}
```

Comment:
This is my opinion:
- Separate `.form` with `.sign-up`. `.form` is an element while `.sign-up` is a block. Different class means different files.
- `.form` should become a reusable component where is consists of `.form__group`, `.form__input`, `.form__actionable`, `.form__button--submit`, etc.
- `.sign-up` should contain block properties such as: `grid`, `background`, `height`, `width`, etc. And it have should it's elements: `.sign-up__header`, `.sign-up__body`, `sign-up__footer`.
- Avoid using color code directly but create a variable for it where it should be resusable. For a better practice, we can group name with similar tones and name it so we don't get confuse with something like `gray-100`, `gray-450` => `map-get($gray, 'lightest')`, `map-get($red, 'blood')`. For color namer, please use (this)[https://chir.ag/projects/name-that-color/#6195ED] so we all get the same name.
- Convert any `px` to `rem`.
```suggestion
// screen/SignUp/styles.scss

.sign-up {
    height: 100%; // if <main> already use 100vh
    background: {
      image: // some image
      position: // some position
      repeat: no-repeat;
      color: map-get($gray, "metallic");
    }
    
    &__header,
    &__body,
    &__actionable,
    &__foooter {
      &:not(:last-child) {
        margin-bottom: rem(24px);
        
        @media-breakpoint-up('md') {
          margin-bottom: rem(32px);
        }
      }
    }
    
    &__header {
      // necessary properties
    }
    
    &__body {
      // necessary properties
    }
    
    &__actionable {
      // necessary properties
    }
    
    &__footer {
      // necessary properties
    }
}

// components/Form/style.scss

.form {
  &__header,
  &__body,
  &__input-group,
  &__actionable,
  &__recaptcha,
  &__footer {
    // some common properties
  }
  
  // further properties (similar implementation as .sign-up)
}

```

Reply:
This should applies to all new `scss` files in this PR such as `nav`.